build/compiler.js

```diff
@@ -7,7 +7,6 @@ const ProgressBarPlugin = require('progress-bar-webpack-plugin');
 const WebpackChunkHash = require('webpack-chunk-hash');
 const webpackDevMiddleware = require('../lib/simple-webpack-dev-middleware');
 const ChunkManifestPlugin = require('./external-chunk-manifest-plugin.js');
-const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
```

Comment:
Don't forget to remove this from the package.json too 😉

Reply:
good point